### ISSUE TITLE
KITTI dataset warning

### ISSUE THREAD

**Issue by ankur7721** [fizyr/keras-retinanet#893]
I have prepared training data in `KITTI` format, while training console reports warning

```
4330/5000 [========================>.....] - ETA: 9:46 - loss: 0.1553 - regression_loss: 0.1429 - classification_loss: 0.0124
preprocessing/generator.py:170: UserWarning: Image with id 572 (shape (555, 1303, 3)) contains the following invalid boxes: [[ 80.08 568.   180.53 592.73]
 [ 81.53 592.73 133.95 618.91]].

```
May i know where it is going wrong ?
I can see that all validations are right

Here's my `label.txt`
```

   A 0.0 0 0.0 13.10 524.36 243.14 549.09 0.0 0.0 0.0 0.0 0.0 0.0 0.0
   B 0.0 0 0.0 80.08 546.18 180.53 570.91 0.0 0.0 0.0 0.0 0.0 0.0 0.0
   C 0.0 0 0.0 80.08 568.00 180.53 592.73 0.0 0.0 0.0 0.0 0.0 0.0 0.0
   D 0.0 0 0.0 81.53 592.73 133.95 618.91 0.0 0.0 0.0 0.0 0.0 0.0 0.0
```

the 15 values represents
```

#Values    Name      Description
----------------------------------------------------------------------------
   1    type         Describes the type of object: 'Car', 'Van', 'Truck',
                     'Pedestrian', 'Person_sitting', 'Cyclist', 'Tram',
                     'Misc' or 'DontCare'
   1    truncated    Float from 0 (non-truncated) to 1 (truncated), where
                     truncated refers to the object leaving image boundaries
   1    occluded     Integer (0,1,2,3) indicating occlusion state:
                     0 = fully visible, 1 = partly occluded
                     2 = largely occluded, 3 = unknown
   1    alpha        Observation angle of object, ranging [-pi..pi]
   4    bbox         2D bounding box of object in the image (0-based index):
                     contains left, top, right, bottom pixel coordinates
   3    dimensions   3D object dimensions: height, width, length (in meters)
   3    location     3D object location x,y,z in camera coordinates (in meters)
   1    rotation_y   Rotation ry around Y-axis in camera coordinates [-pi..pi]
   1    score        Only for results: Float, indicating confidence in
                     detection, needed for p/r curves, higher is better.
```

**Comment by hgaiser** [comment]
The warning indicates that you have a bounding box with `y_min = 568` and `y_max = 592.73`, but your image height is `555`. This is out of bounds for the image, so it can't use this annotation. Perhaps your `label.txt` is defined as `y_min, x_min, y_max, x_max`? In keras-retinanet we define it as `x_min, y_min, x_max, y_max`.

Closing this as it appears there is no issue from our code base here.
